data/netcdf_data.py

```diff
@@ -691,11 +691,11 @@ def yx_dimensions(self) -> Tuple[str, str]:
 
         return y_dim, x_dim
 
-    def get_dataset_variable(self, key: str):
+    def get_dataset_variable(self, key: str) -> xarray.DataArray:
         """
-        Returns the value of a given variable name from the dataset
+        Returns the xarray.DataArray for a given variable key
         """
-        return self.dataset.variables[key]
```

Comment:
Pretty sure I caused this when I migrated from the python `netcdf4` library to `xarray` like 3 years ago :joy_cat: